mono-traversable/src/Data/MonoTraversable.hs

```diff
@@ -1140,6 +1142,9 @@ instance MonoPointed S.ByteString where
 instance MonoPointed L.ByteString where
     opoint = L.singleton
     {-# INLINE opoint #-}
+instance MonoPointed B.Builder where
```

Comment:
LGTM. Can you add a minor version bump, changelog entry, and a `@since` comment?